src/index.d.ts

```diff
@@ -33,7 +33,7 @@ export * from './sequelize';
 export { Sequelize as default } from './sequelize';
 export { useInflection } from './utils/string';
 export { isModelStatic, isSameInitialModel } from './utils/model-utils';
-export type { Validator } from './utils/validator-extras';
+export type { validator } from './utils/validator-extras';
```

Comment:
This change is not necessary, I recommend to keep the previous casing to avoid an unnecessary breaking change